lib/fog/openstack/identity.rb

```diff
@@ -35,14 +39,6 @@ def initialize(options = {})
           @openstack_identity_prefix = options[:openstack_identity_prefix]
```

Comment:
Yeah, more likely, I haven't done a full clean-up yet. Good catch, I'll remove it. Thanks